### ISSUE TITLE
Throw if file not found

### ISSUE THREAD

**Issue by Amatewasu** [rhys-vdw/ts-auto-guard#234]
Currently, if we run `ts-auto-guard my-file.ts` and `my-file.ts` is not found, ts-auto-guard output `Done!`.

It would be nice to display that the file has not been found.

**Comment by rhys-vdw** [comment]
@Amatewasu good idea. Would you like to open a PR? I'm sure it would be a simple fix.

Looks like it once already worked:
https://github.com/rhys-vdw/ts-auto-guard/blob/c6c97c25b50e8b00d582f864b14c20e36fa23634/src/cli.ts#L128

In any case, you'll probably need to loop through the files and throw here:
https://github.com/rhys-vdw/ts-auto-guard/blob/c6c97c25b50e8b00d582f864b14c20e36fa23634/src/index.ts#L1047
